e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/ConstraintsTests.java

```diff
@@ -478,8 +491,39 @@ void compilationFailsOutdatedSimulationNoSimDataset() throws IOException {
       assertEquals(false,constraintRepsonse.success());
       assertEquals(1,constraintRepsonse.errors().size());
       final ConstraintError error = constraintRepsonse.errors().get(0);
-      assertEquals("Constraint 'fruit_equal_peel': TypeError: TS2345 Argument of type '\"/my_boolean\"' is not assignable to parameter of type 'ResourceName'.",error.message());
+      assertEquals("Constraint 'fruit_equal_peel' compilation failed:\n"
+                   + " TypeError: TS2345 Argument of type '\"/my_boolean\"' is not assignable to parameter of type 'ResourceName'.",error.message());
+
+    }
+
+    @Test
+    @DisplayName("if a constraint is updated and then reverted, Constraints will load the cached value from before the update.")
+    void constraintInvalidModification() throws IOException {
```

Comment:
Minor: Could you please add a couple brief comments in the same vein as how the rest of the tests are commented?

Reply:
Added comments to this test case and a brief description